tests/test_model_methods.py

```diff
@@ -749,6 +749,21 @@ def test_skip_nodata_reservoir(clipped_wflow_model):
         )
 
 
+def test_setup_lulc_sed(example_sediment_model, planted_forest_testdata):
+    example_sediment_model.setup_lulcmaps(
+        lulc_fn="globcover",
+        planted_forest_fn=planted_forest_testdata,
+        lulc_vars=["USLE_C"],
+        planted_forest_c=0.0881,
+        orchard_name="Orchard",
+        orchard_c=0.2188,
+    )
+    da = example_sediment_model.grid["USLE_C"].raster.sample(
+        planted_forest_testdata.geometry.centroid
+    )
+    assert np.all(da.values == np.array([0.0881, 0.2188]))
```

Comment:
Maybe convert to integers? Testing against floats can be unwanted.

Reply:
These are the "mapping" values as requested in the argument of my function. So I really want them to be exactly what I asked for here.

Reply:
You could multiply it by e.g. 1e6 first to keep some precision.